<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(C:/Users/yehia_xd1awm4/Downloads/360_F_327515607_Hcps04aaEc7Ki43d1XZPxwcv0ZaIaorh.jpg);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Subject:</string>
    </property>
   </widget>
   <widget class="QLabel" name="attachments">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>680</y>
      <width>731</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold; /* Makes the text bold */
</string>
    </property>
    <property name="text">
     <string>Attachments:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="toText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>19</y>
      <width>691</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>750</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(C:/Users/yehia_xd1awm4/Downloads/neumorphic-rectangle-button-free-png.jpeg);
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Send Mail</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="subjectText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>561</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 10px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="attachButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(C:/Users/yehia_xd1awm4/Downloads/neumorphic-rectangle-button-free-png.jpeg);
	color: white;
}</string>
    </property>
    <property name="text">
     <string>Attachment</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="mailText">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>751</width>
      <height>441</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid black;
border-radius: 15px; /* Adjust the value to change the roundness */
background-image: url(C:/Users/yehia_xd1awm4/Downloads/blank-white-background-xbsfzsltjksfompa.jpg);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: black; /* Sets the text color to blue */
font-size: 14px; /* Sets the font size to 14 pixels */
font-weight: bold;</string>
    </property>
    <property name="text">
     <string>Mail text</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border-image: url(C:/Users/yehia_xd1awm4/Downloads/images.png);
	color: white;


}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
